src/classes/actions-cache/http-responses.ts

```diff
@@ -0,0 +1,19 @@
+import {TypedResponse} from '@actions/http-client/lib/interfaces';
+import {HttpClientError} from '@actions/http-client';
+
+export const isSuccessStatusCode = (statusCode?: number): boolean => {
```

Comment:
The code is inherited from the `@toolkit/cache` and i prefer to keep it unchanged because the API is undocumented and i'd prefer to avoid the risks. The whole folder `actions-cache` is the subject of the refactoring anyway